docs/AdditionalArguments.md

```diff
@@ -0,0 +1,25 @@
+# Passing additional arguments through commandline
+You are here because you are looking for syntax & details to pass additional arguments to either `vstest.console.exe` or `dotnet test` 
+
+Additional arguments is a way to pass arguments as [name]=[value] with a separator `-- `. Note the space after --. This can be used to pass additional arguments to test extensions like test adapters. They will be available to the test extensions as part of `runsettings`. 
```

Comment:
``They will be available to the test extensions as part of `runsettings`. ``
Updated arguments is available to all who uses runsettings and not only to extensions.